main.js

```diff
@@ -0,0 +1,160 @@
+        description: 'A frame drum for people who like to jig.',
+        details: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.',
+        origin: 'Ireland',
+        imageUrl: 'https://goo.gl/YNb74O'
+      },
+      {
+        itemId: 8,
+        name: 'TR-09',
+        brand: 'Roland',
+        price: 399,
+        description: 'A drum machine for people with no desk space.',
+        details: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.',
+        origin: 'Japan',
+        imageUrl: 'https://goo.gl/gJYUxz'
+      }
+    ]
+  },
+  details: {
+    item: null
+  }
+}
+
+function renderItem(item) {
+  var $item = document.createElement('div')
+  $item.classList.add('card')
+  $item.classList.add('text-center')
+  $item.setAttribute('style', 'width: 15rem')
+  $item.setAttribute('style', 'height: 30rem')
+  var $img = document.createElement('img')
+  $img.classList.add('card-img-top')
+  $img.setAttribute('src', item.imageUrl)
+  var $itemBody = document.createElement('div')
+  $itemBody.classList.add('card-body')
+  var $name = document.createElement('h5')
+  $name.classList.add('card-title')
+  $name.textContent = item.name
+  var $brand = document.createElement('h6')
+  $brand.classList.add('card-subtitle')
+  $brand.textContent = item.brand
+  var $price = document.createElement('h6')
+  $price.classList.add('card-text')
+  $price.textContent = '$' + item.price
+  $item.appendChild($img)
+  $item.appendChild($itemBody)
+  $itemBody.appendChild($name)
+  $itemBody.appendChild($brand)
+  $itemBody.appendChild($price)
+  return $item
+}
+
+function renderCatalog(catalog) {
+  var $container = document.createElement('div')
+  $container.classList.add('container')
+  var $heading = document.createElement('h1')
+  $heading.textContent = 'Jamazon'
+  $heading.classList.add('text-center')
+  var $row = document.createElement('div')
+  $row.classList.add('row')
+  var $rowTwo = document.createElement('div')
+  $rowTwo.classList.add('row')
+  var equalWidth = document.createElement('div')
+  equalWidth.classList.add('w-100')
+  $container.appendChild($heading)
+  $container.appendChild($row)
+  $container.appendChild($rowTwo)
+
+  for (var i = 0; i < 4; i++) {
```

Comment:
You can put all of the columns in one `$row`. Use one or more on each `$col`. Bootstrap uses media queries to automatically move extra columns down the page.
https://getbootstrap.com/docs/4.1/layout/grid/#mix-and-match